src/cookiebot.jsx

```diff
@@ -1,8 +1,13 @@
 import CookieBot from 'react-cookiebot';
+import { useSelector } from 'react-redux';
 
 import config from '@plone/volto/registry';
 
 export const CookieBotAppExtra = (props) => {
   const domainGroupId = config.settings.cookiebotDomainGroupId;
-  return <CookieBot domainGroupId={domainGroupId} />;
+  const language = useSelector((state) => state.intl.locale);
+
+  return (
+    <CookieBot domainGroupId={domainGroupId} language={language || undefined} />
```

Comment:
we should add a default language instead of leaving as undefined

Reply:
I have added en as default language @ionlizarazu